src/transactions/sendBtcTransaction.ts

```diff
@@ -1,10 +1,15 @@
 import { createUnsecuredToken, Json } from 'jsontokens';
 import { BitcoinNetwork } from '../provider';
 
+export interface Recipient {
+  address: string;
+  amountSats: number;
```

Comment:
Should we use bigint here instead? Number allows decimals.

Reply:
sometimes bignumber does not work properly when passed in props. while implementing, i tried setting the amountSats property as bignumber in sats connect but in the extension it gave me an error `TypeError: amountSats.toNumber is not a function` which basically shows that bignumber type was not being recognised. I discussed with haseeb and we decided to use amount as type and later convert it in the extension and app.